### PR TITLE
Mvp rename package

### LINKED ISSUE #43: MVP - Rename to ResearchRoster
Replacing AB3 with ResearchRoster
Renaming package *
Replace AB3 to ReasearchRoster *

### PR DESCRIPTION
**Pull request by PrishaVP**
fixes #43

### CODE REVIEW

File: src/main/java/seedu/researchroster/logic/Logic.java [hunk 1]
```diff
@@ -24,17 +24,17 @@ public interface Logic {
...
      */
-    ReadOnlyAddressBook getAddressBook();
+    ReadOnlyResearchRoster getAddressBook();
 
     /** Returns an unmodifiable view of the filtered list of persons */
     ObservableList<Person> getFilteredPersonList();
 
     /**
-     * Returns the user prefs' address book file path.
+     * Returns the user prefs' researchroster book file path.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/Logic.java:37]:
```suggestion
     * Returns the user prefs' ResearchRoster book file path.
```

File: src/main/java/seedu/researchroster/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-import seedu.address.model.person.Person;
+import seedu.researchroster.commons.util.ToStringBuilder;
+import seedu.researchroster.logic.Messages;
+import seedu.researchroster.logic.commands.exceptions.CommandException;
+import seedu.researchroster.model.Model;
+import seedu.researchroster.model.person.Person;
 
 /**
- * Adds a person to the address book.
+ * Adds a person to the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/commands/AddCommand.java:17]:
```suggestion
 * Adds a person to the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/logic/commands/AddCommand.java [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
  */
 public class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the researchroster book. "
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/commands/AddCommand.java:23]:
```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the ResearchRoster book. "
```

File: src/main/java/seedu/researchroster/logic/commands/AddCommand.java [hunk 4]
```diff
@@ -36,7 +36,7 @@ public class AddCommand extends Command {
             + PREFIX_TAG + "owesMoney";
 
     public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the researchroster book";
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/commands/AddCommand.java:39]:
```suggestion
    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the ResearchRoster book";
```

File: src/main/java/seedu/researchroster/logic/commands/ClearCommand.java [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
 import static java.util.Objects.requireNonNull;
 
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
+import seedu.researchroster.model.Model;
+import seedu.researchroster.model.ResearchRoster;
 
 /**
- * Clears the address book.
+ * Clears the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/commands/ClearCommand.java:9]:
```suggestion
 * Clears the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/logic/commands/DeleteCommand.java [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
+import seedu.researchroster.commons.core.index.Index;
+import seedu.researchroster.commons.util.ToStringBuilder;
+import seedu.researchroster.logic.Messages;
+import seedu.researchroster.logic.commands.exceptions.CommandException;
+import seedu.researchroster.model.Model;
+import seedu.researchroster.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a person identified using it's displayed index from the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/commands/DeleteCommand.java:15]:
```suggestion
 * Deletes a person identified using it's displayed index from the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/logic/commands/EditCommand.java [hunk 7]
```diff
@@ -15,21 +15,21 @@
...
+import seedu.researchroster.model.person.Address;
+import seedu.researchroster.model.person.Email;
+import seedu.researchroster.model.person.Name;
+import seedu.researchroster.model.person.Person;
+import seedu.researchroster.model.person.Phone;
+import seedu.researchroster.model.tag.Tag;
 
 /**
- * Edits the details of an existing person in the address book.
+ * Edits the details of an existing person in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/commands/EditCommand.java:32]:
```suggestion
 * Edits the details of an existing person in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/logic/commands/EditCommand.java [hunk 8]
```diff
@@ -50,7 +50,7 @@ public class EditCommand extends Command {
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the researchroster book.";
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/commands/EditCommand.java:53]:
```suggestion
    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the ResearchRoster book.";
```

File: src/main/java/seedu/researchroster/logic/commands/FindCommand.java [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
-import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.researchroster.commons.util.ToStringBuilder;
+import seedu.researchroster.logic.Messages;
+import seedu.researchroster.model.Model;
+import seedu.researchroster.model.person.NameContainsKeywordsPredicate;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Finds and lists all persons in researchroster book whose name contains any of the argument keywords.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/commands/FindCommand.java:11]:
```suggestion
 * Finds and lists all persons in ResearchRoster book whose name contains any of the argument keywords.
```

File: src/main/java/seedu/researchroster/logic/commands/ListCommand.java [hunk 10]
```diff
@@ -1,12 +1,12 @@
...
 import static java.util.Objects.requireNonNull;
-import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+import static seedu.researchroster.model.Model.PREDICATE_SHOW_ALL_PERSONS;
 
-import seedu.address.model.Model;
+import seedu.researchroster.model.Model;
 
 /**
- * Lists all persons in the address book to the user.
+ * Lists all persons in the researchroster book to the user.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/commands/ListCommand.java:9]:
```suggestion
 * Lists all persons in the ResearchRoster book to the user.
```

File: src/main/java/seedu/researchroster/logic/parser/ParserUtil.java [hunk 11]
```diff
@@ -66,10 +66,10 @@ public static Phone parsePhone(String phone) throws ParseException {
     }
 
     /**
-     * Parses a {@code String address} into an {@code Address}.
+     * Parses a {@code String researchroster} into an {@code Address}.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/parser/ParserUtil.java:69]:
```suggestion
     * Parses a {@code String ResearchRoster} into an {@code Address}.
```

File: src/main/java/seedu/researchroster/logic/parser/ParserUtil.java [hunk 12]
```diff
@@ -66,10 +66,10 @@ public static Phone parsePhone(String phone) throws ParseException {
...
      * Leading and trailing whitespaces will be trimmed.
      *
-     * @throws ParseException if the given {@code address} is invalid.
+     * @throws ParseException if the given {@code researchroster} is invalid.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/logic/parser/ParserUtil.java:72]:
```suggestion
     * @throws ParseException if the given {@code ResearchRoster} is invalid.
```

File: src/main/java/seedu/researchroster/model/Model.java [hunk 13]
```diff
@@ -35,44 +35,45 @@ public interface Model {
     void setGuiSettings(GuiSettings guiSettings);
 
     /**
-     * Returns the user prefs' address book file path.
+     * Returns the user prefs' researchroster book file path.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/Model.java:38]:
```suggestion
     * Returns the user prefs' ResearchRoster book file path.
```

File: src/main/java/seedu/researchroster/model/Model.java [hunk 14]
```diff
@@ -35,44 +35,45 @@ public interface Model {
...
      */
     Path getAddressBookFilePath();
 
     /**
-     * Sets the user prefs' address book file path.
+     * Sets the user prefs' researchroster book file path.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/Model.java:43]:
```suggestion
     * Sets the user prefs' ResearchRoster book file path.
```

File: src/main/java/seedu/researchroster/model/Model.java [hunk 15]
```diff
@@ -35,44 +35,45 @@ public interface Model {
...
      */
     void setAddressBookFilePath(Path addressBookFilePath);
 
     /**
-     * Replaces address book data with the data in {@code addressBook}.
+     * Replaces researchroster book data with the data in {@code addressBook}.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/Model.java:48]:
```suggestion
     * Replaces ResearchRoster book data with the data in {@code addressBook}.
```

File: src/main/java/seedu/researchroster/model/Model.java [hunk 16]
```diff
@@ -35,44 +35,45 @@ public interface Model {
...
+    void setAddressBook(ReadOnlyResearchRoster addressBook);
 
-    /** Returns the AddressBook */
-    ReadOnlyAddressBook getAddressBook();
+    /** Returns the ResearchRoster */
+    ReadOnlyResearchRoster getAddressBook();
 
     /**
-     * Returns true if a person with the same identity as {@code person} exists in the address book.
+     * Returns true if a person with the same identity as {@code person} exists in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/Model.java:56]:
```suggestion
     * Returns true if a person with the same identity as {@code person} exists in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/model/Model.java [hunk 17]
```diff
@@ -35,44 +35,45 @@ public interface Model {
...
      */
     boolean hasPerson(Person person);
 
     /**
      * Deletes the given person.
-     * The person must exist in the address book.
+     * The person must exist in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/Model.java:62]:
```suggestion
     * The person must exist in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/model/Model.java [hunk 18]
```diff
@@ -35,44 +35,45 @@ public interface Model {
...
      */
     void deletePerson(Person target);
 
     /**
      * Adds the given person.
-     * {@code person} must not already exist in the address book.
+     * {@code person} must not already exist in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/Model.java:68]:
```suggestion
     * {@code person} must not already exist in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/model/Model.java [hunk 19]
```diff
@@ -35,44 +35,45 @@ public interface Model {
...
      */
     void addPerson(Person person);
 
     /**
      * Replaces the given person {@code target} with {@code editedPerson}.
-     * {@code target} must exist in the address book.
-     * The person identity of {@code editedPerson} must not be the same as another existing person in the address book.
+     * {@code target} must exist in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/Model.java:74]:
```suggestion
     * {@code target} must exist in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/model/Model.java [hunk 20]
```diff
@@ -35,44 +35,45 @@ public interface Model {
...
+     * The person identity of {@code editedPerson} must not be the
+     * same as another existing person in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/Model.java:76]:
```suggestion
     * same as another existing person in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/model/ModelManager.java [hunk 21]
```diff
@@ -1,43 +1,43 @@
...
-import seedu.address.commons.core.GuiSettings;
-import seedu.address.commons.core.LogsCenter;
-import seedu.address.model.person.Person;
+import seedu.researchroster.commons.core.GuiSettings;
+import seedu.researchroster.commons.core.LogsCenter;
+import seedu.researchroster.model.person.Person;
 
 /**
- * Represents the in-memory model of the address book data.
+ * Represents the in-memory model of the researchroster book data.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ModelManager.java:17]:
```suggestion
 * Represents the in-memory model of the ResearchRoster book data.
```

File: src/main/java/seedu/researchroster/model/ModelManager.java [hunk 22]
```diff
@@ -1,43 +1,43 @@
...
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
 
-    private final AddressBook addressBook;
+    private final ResearchRoster researchRoster;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
 
     /**
-     * Initializes a ModelManager with the given addressBook and userPrefs.
+     * Initializes a ModelManager with the given researchRoster and userPrefs.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ModelManager.java:27]: ****
```suggestion
     * Initializes a ModelManager with the givenResearchRoster and userPrefs.
```

File: src/main/java/seedu/researchroster/model/ModelManager.java [hunk 23]
```diff
@@ -1,43 +1,43 @@
...
      */
-    public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs) {
+    public ModelManager(ReadOnlyResearchRoster addressBook, ReadOnlyUserPrefs userPrefs) {
         requireAllNonNull(addressBook, userPrefs);
 
-        logger.fine("Initializing with address book: " + addressBook + " and user prefs " + userPrefs);
+        logger.fine("Initializing with researchroster book: " + addressBook + " and user prefs " + userPrefs);
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ModelManager.java:32]:
```suggestion
        logger.fine("Initializing with ResearchRoster book: " + addressBook + " and user prefs " + userPrefs);
```

File: src/main/java/seedu/researchroster/model/ReadOnlyResearchRoster.java [hunk 24]
```diff
@@ -1,12 +1,12 @@
-package seedu.address.model;
+package seedu.researchroster.model;
 
 import javafx.collections.ObservableList;
-import seedu.address.model.person.Person;
+import seedu.researchroster.model.person.Person;
 
 /**
- * Unmodifiable view of an address book
+ * Unmodifiable view of an researchroster book
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ReadOnlyResearchRoster.java:7]:
```suggestion
 * Unmodifiable view of an ResearchRoster book
```

File: src/main/java/seedu/researchroster/model/ResearchRoster.java [hunk 25]
```diff
@@ -1,19 +1,19 @@
...
-import seedu.address.commons.util.ToStringBuilder;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.UniquePersonList;
+import seedu.researchroster.commons.util.ToStringBuilder;
+import seedu.researchroster.model.person.Person;
+import seedu.researchroster.model.person.UniquePersonList;
 
 /**
- * Wraps all data at the address-book level
+ * Wraps all data at the researchroster-book level
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ResearchRoster.java:13]:
```suggestion
 * Wraps all data at the ResearchRoster-book level
```

File: src/main/java/seedu/researchroster/model/ResearchRoster.java [hunk 26]
```diff
@@ -60,25 +60,26 @@ public void resetData(ReadOnlyAddressBook newData) {
     //// person-level operations
 
     /**
-     * Returns true if a person with the same identity as {@code person} exists in the address book.
+     * Returns true if a person with the same identity as {@code person} exists in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ResearchRoster.java:63]:
```suggestion
     * Returns true if a person with the same identity as {@code person} exists in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/model/ResearchRoster.java [hunk 27]
```diff
@@ -60,25 +60,26 @@ public void resetData(ReadOnlyAddressBook newData) {
...
      */
     public boolean hasPerson(Person person) {
         requireNonNull(person);
         return persons.contains(person);
     }
 
     /**
-     * Adds a person to the address book.
-     * The person must not already exist in the address book.
+     * Adds a person to the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ResearchRoster.java:71]:
```suggestion
     * Adds a person to the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/model/ResearchRoster.java [hunk 28]
```diff
@@ -60,25 +60,26 @@ public void resetData(ReadOnlyAddressBook newData) {
...
+     * The person must not already exist in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ResearchRoster.java:72]:
```suggestion
     * The person must not already exist in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/model/ResearchRoster.java [hunk 29]
```diff
@@ -60,25 +60,26 @@ public void resetData(ReadOnlyAddressBook newData) {
...
      */
     public void addPerson(Person p) {
         persons.add(p);
     }
 
     /**
      * Replaces the given person {@code target} in the list with {@code editedPerson}.
-     * {@code target} must exist in the address book.
-     * The person identity of {@code editedPerson} must not be the same as another existing person in the address book.
+     * {@code target} must exist in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ResearchRoster.java:80]:
```suggestion
     * {@code target} must exist in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/model/ResearchRoster.java [hunk 30]
```diff
@@ -60,25 +60,26 @@ public void resetData(ReadOnlyAddressBook newData) {
...
+     * The person identity of {@code editedPerson} must not be the same as another
+     * existing person in the researchroster book.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/model/ResearchRoster.java:82]:
```suggestion
     * existing person in the ResearchRoster book.
```

File: src/main/java/seedu/researchroster/storage/JsonSerializableResearchRoster.java [hunk 31]
```diff
@@ -0,0 +1,60 @@
...
+     * Converts a given {@code ReadOnlyResearchRoster} into this class for Jackson use.
+     *
+     * @param source future changes to this will not affect the created {@code JsonSerializableResearchRoster}.
+     */
+    public JsonSerializableResearchRoster(ReadOnlyResearchRoster source) {
+        persons.addAll(source.getPersonList().stream().map(JsonAdaptedPerson::new).collect(Collectors.toList()));
+    }
+
+    /**
+     * Converts this researchroster book into the model's {@code ResearchRoster} object.
```
Comment (PrishaVP) [inline src/main/java/seedu/researchroster/storage/JsonSerializableResearchRoster.java:44]:
```suggestion
     * Converts this ResearchRoster book into the model's {@code ResearchRoster} object.
```

File: src/test/java/seedu/researchroster/logic/commands/CommandTestUtil.java [hunk 32]
```diff
@@ -99,21 +99,21 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
...
 
         assertThrows(CommandException.class, expectedMessage, () -> command.execute(actualModel));
-        assertEquals(expectedAddressBook, actualModel.getAddressBook());
+        assertEquals(expectedResearchRoster, actualModel.getAddressBook());
         assertEquals(expectedFilteredList, actualModel.getFilteredPersonList());
     }
     /**
      * Updates {@code model}'s filtered list to show only the person at the given {@code targetIndex} in the
-     * {@code model}'s address book.
+     * {@code model}'s researchroster book.
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/commands/CommandTestUtil.java:116]:
```suggestion
     * {@code model}'s ResearchRoster.
```

File: src/test/java/seedu/researchroster/logic/commands/DeleteCommandTest.java [hunk 33]
```diff
@@ -71,7 +71,7 @@ public void execute_invalidIndexFilteredList_throwsCommandException() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
 
         Index outOfBoundIndex = INDEX_SECOND_PERSON;
-        // ensures that outOfBoundIndex is still in bounds of address book list
+        // ensures that outOfBoundIndex is still in bounds of researchroster book list
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/commands/DeleteCommandTest.java:74]:
```suggestion
        // ensures that outOfBoundIndex is still in bounds of ResearchRoster list
```

File: src/test/java/seedu/researchroster/logic/commands/EditCommandTest.java [hunk 34]
```diff
@@ -112,7 +112,7 @@ public void execute_duplicatePersonUnfilteredList_failure() {
     public void execute_duplicatePersonFilteredList_failure() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
 
-        // edit person in filtered list into a duplicate in address book
+        // edit person in filtered list into a duplicate in researchroster book
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/commands/EditCommandTest.java:115]:
```suggestion
        // edit person in filtered list into a duplicate in ResearchRoster
```

File: src/test/java/seedu/researchroster/logic/commands/EditCommandTest.java [hunk 35]
```diff
@@ -131,13 +131,13 @@ public void execute_invalidPersonIndexUnfilteredList_failure() {
 
     /**
      * Edit filtered list where index is larger than size of filtered list,
-     * but smaller than size of address book
+     * but smaller than size of researchroster book
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/commands/EditCommandTest.java:134]:
```suggestion
     * but smaller than size of ResearchRoster
```

File: src/test/java/seedu/researchroster/logic/commands/EditCommandTest.java [hunk 36]
```diff
@@ -131,13 +131,13 @@ public void execute_invalidPersonIndexUnfilteredList_failure() {
...
      */
     @Test
     public void execute_invalidPersonIndexFilteredList_failure() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
         Index outOfBoundIndex = INDEX_SECOND_PERSON;
-        // ensures that outOfBoundIndex is still in bounds of address book list
+        // ensures that outOfBoundIndex is still in bounds of researchroster book list
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/commands/EditCommandTest.java:140]:
```suggestion
        // ensures that outOfBoundIndex is still in bounds of ResearchRoster list
```

File: src/test/java/seedu/researchroster/logic/commands/EditPersonDescriptorTest.java [hunk 37]
```diff
@@ -48,7 +48,7 @@ public void equals() {
         editedAmy = new EditPersonDescriptorBuilder(DESC_AMY).withEmail(VALID_EMAIL_BOB).build();
         assertFalse(DESC_AMY.equals(editedAmy));
 
-        // different address -> returns false
+        // different researchroster -> returns false
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/commands/EditPersonDescriptorTest.java:51]:
```suggestion
        // different // different address -> returns false
```

File: src/test/java/seedu/researchroster/logic/commands/EditPersonDescriptorTest.java [hunk 38]
```diff
@@ -63,7 +63,7 @@ public void toStringMethod() {
         String expected = EditPersonDescriptor.class.getCanonicalName() + "{name="
                 + editPersonDescriptor.getName().orElse(null) + ", phone="
                 + editPersonDescriptor.getPhone().orElse(null) + ", email="
-                + editPersonDescriptor.getEmail().orElse(null) + ", address="
+                + editPersonDescriptor.getEmail().orElse(null) + ", researchroster="
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/commands/EditPersonDescriptorTest.java:66]:
```suggestion
                + editPersonDescriptor.getEmail().orElse(null) + ", address="
```

File: src/test/java/seedu/researchroster/logic/parser/AddCommandParserTest.java [hunk 39]
```diff
@@ -106,7 +106,7 @@ public void parse_repeatedNonTagValue_failure() {
         assertParseFailure(parser, INVALID_PHONE_DESC + validExpectedPersonString,
                 Messages.getErrorMessageForDuplicatePrefixes(PREFIX_PHONE));
 
-        // invalid address
+        // invalid researchroster
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/parser/AddCommandParserTest.java:109]:
```suggestion
        // invalid address
```

File: src/test/java/seedu/researchroster/logic/parser/AddCommandParserTest.java [hunk 40]
```diff
@@ -124,7 +124,7 @@ public void parse_repeatedNonTagValue_failure() {
         assertParseFailure(parser, validExpectedPersonString + INVALID_PHONE_DESC,
                 Messages.getErrorMessageForDuplicatePrefixes(PREFIX_PHONE));
 
-        // invalid address
+        // invalid researchroster
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/parser/AddCommandParserTest.java:127]:
```suggestion
        // invalid address
```

File: src/test/java/seedu/researchroster/logic/parser/AddCommandParserTest.java [hunk 41]
```diff
@@ -153,7 +153,7 @@ public void parse_compulsoryFieldMissing_failure() {
         assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + VALID_EMAIL_BOB + ADDRESS_DESC_BOB,
                 expectedMessage);
 
-        // missing address prefix
+        // missing researchroster prefix
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/parser/AddCommandParserTest.java:156]:
```suggestion
        // missing address prefix
```

File: src/test/java/seedu/researchroster/logic/parser/AddCommandParserTest.java [hunk 42]
```diff
@@ -176,7 +176,7 @@ public void parse_invalidValue_failure() {
         assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + INVALID_EMAIL_DESC + ADDRESS_DESC_BOB
                 + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Email.MESSAGE_CONSTRAINTS);
 
-        // invalid address
+        // invalid researchroster
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/parser/AddCommandParserTest.java:179]:
```suggestion
        // invalid address
```

File: src/test/java/seedu/researchroster/logic/parser/EditCommandParserTest.java [hunk 43]
```diff
@@ -86,7 +86,7 @@ public void parse_invalidValue_failure() {
         assertParseFailure(parser, "1" + INVALID_NAME_DESC, Name.MESSAGE_CONSTRAINTS); // invalid name
         assertParseFailure(parser, "1" + INVALID_PHONE_DESC, Phone.MESSAGE_CONSTRAINTS); // invalid phone
         assertParseFailure(parser, "1" + INVALID_EMAIL_DESC, Email.MESSAGE_CONSTRAINTS); // invalid email
-        assertParseFailure(parser, "1" + INVALID_ADDRESS_DESC, Address.MESSAGE_CONSTRAINTS); // invalid address
+        assertParseFailure(parser, "1" + INVALID_ADDRESS_DESC, Address.MESSAGE_CONSTRAINTS); // invalid researchroster
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/parser/EditCommandParserTest.java:89]:
```suggestion
        assertParseFailure(parser, "1" + INVALID_ADDRESS_DESC, Address.MESSAGE_CONSTRAINTS); // invalid address
```

File: src/test/java/seedu/researchroster/logic/parser/EditCommandParserTest.java [hunk 44]
```diff
@@ -150,7 +150,7 @@ public void parse_oneFieldSpecified_success() {
         expectedCommand = new EditCommand(targetIndex, descriptor);
         assertParseSuccess(parser, userInput, expectedCommand);
 
-        // address
+        // researchroster
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/logic/parser/EditCommandParserTest.java:153]:
```suggestion
        // address
```

File: src/test/java/seedu/researchroster/model/person/AddressTest.java [hunk 45]
```diff
@@ -21,7 +21,7 @@ public void constructor_invalidAddress_throwsIllegalArgumentException() {
 
     @Test
     public void isValidAddress() {
-        // null address
+        // null researchroster
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/model/person/AddressTest.java:24]:
```suggestion
        // null address
```

File: src/test/java/seedu/researchroster/model/person/NameContainsKeywordsPredicateTest.java [hunk 46]
```diff
@@ -68,7 +68,7 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("Carol"));
         assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
 
-        // Keywords match phone, email and address, but does not match name
+        // Keywords match phone, email and researchroster, but does not match name
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/model/person/NameContainsKeywordsPredicateTest.java:71]:
```suggestion
        // Keywords match phone, email and address, but does not match name
```

File: src/test/java/seedu/researchroster/model/person/PersonTest.java [hunk 47]
```diff
@@ -81,7 +81,7 @@ public void equals() {
         editedAlice = new PersonBuilder(ALICE).withEmail(VALID_EMAIL_BOB).build();
         assertFalse(ALICE.equals(editedAlice));
 
-        // different address -> returns false
+        // different researchroster -> returns false
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/model/person/PersonTest.java:84]:
```suggestion
        // different address -> returns false
```

File: src/test/java/seedu/researchroster/model/person/PersonTest.java [hunk 48]
```diff
@@ -93,7 +93,8 @@ public void equals() {
     @Test
     public void toStringMethod() {
         String expected = Person.class.getCanonicalName() + "{name=" + ALICE.getName() + ", phone=" + ALICE.getPhone()
-                + ", email=" + ALICE.getEmail() + ", address=" + ALICE.getAddress() + ", tags=" + ALICE.getTags() + "}";
+                + ", email=" + ALICE.getEmail() + ", researchroster=" + ALICE.getAddress()
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/model/person/PersonTest.java:96]:
```suggestion
                + ", email=" + ALICE.getEmail() + ", address=" + ALICE.getAddress()
```

File: src/test/java/seedu/researchroster/testutil/ResearchRosterBuilder.java [hunk 49]
```diff
@@ -0,0 +1,34 @@
+package seedu.researchroster.testutil;
+
+import seedu.researchroster.model.ResearchRoster;
+import seedu.researchroster.model.person.Person;
+
+/**
+ * A utility class to help with building Addressbook objects.
```
Comment (PrishaVP) [inline src/test/java/seedu/researchroster/testutil/ResearchRosterBuilder.java:7]:
```suggestion
 * A utility class to help with building ResearchRoster objects.
```

**Review by PrishaVP — COMMENTED** [review]
Change lower caps to upper. Have not finished fixing all problems

**Comment by wkxcass** [comment]
- Perhaps the changes above are not really needed, as we are essentially still working with a model of an address book... (plus some changes affected the intended meaning of some statements, e.g., a person's address -> a person's researchroster) what do the rest think!
- Suggestion: can instead change the app's name displayed on our GUI (the one that'll appear on the top of the window)!

**Pull request closed without merging**